doc/Building.md

```diff
@@ -0,0 +1,199 @@
+# Setting up a build environment for KOReader
+
+These instructions are intended to build the emulator in Linux and MacOS. Windows users are suggested to develop in a [Linux VM](https://www.howtogeek.com/howto/11287/how-to-run-ubuntu-in-windows-7-with-vmware-player/) or using the [Windows Subsystem for Linux](https://en.wikipedia.org/wiki/Windows_Subsystem_for_Linux).
```

Comment:
WSL was more thinking aloud. I'm not sure we should recommend using WSL unless someone's actually verified it works for the purpose? The AppImage runs in WSL so logic dictates it should, but... :-)

Not that I've verified VMWare player personally or anything, but it's Ubuntu ergo it will work as expected.

Since I happen to be typing in Windows right now I suppose I'll give [this](https://docs.microsoft.com/en-us/windows/wsl/wsl2-install) a go, but it wants to restart after executing the first command and I don't want to restart. :-P